config/development.yaml

```diff
@@ -38,4 +38,11 @@ session:
 e2e:
   userTestPassword: ''
 appInsights:
-  instrumentationKey: false
\ No newline at end of file
+  instrumentationKey: false
+featureToggles:
+  c100Rebuild: C100_REBUILD_FLAG
```

Comment:
Same here as well, we an enable it by default in Dev env I think

Reply:
Done.